middlewares/reload.ts

```diff
@@ -1,3 +1,5 @@
+import { encodeBase64 } from "jsr:@std/encoding@1.0.5/base64";
```

Comment:
Please, import the function from https://github.com/lumeland/lume/blob/main/deps/base64.ts

Reply:
Thanks! Fixed